src/hooks/coreSDK/useCustomNetworkContracts.ts

```diff
@@ -0,0 +1,32 @@
+import type { ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import { useCurrentChain } from '../useChains'
+import type { ContractNetworkConfig, ContractNetworksConfig } from '@safe-global/protocol-kit'
+import { IS_OFFICIAL_HOST, IS_PRODUCTION } from '@/config/constants'
+
+const isContractNetworkConfig = (obj: Omit<ChainInfo['contractAddresses'], 'safeWebAuthnSignerFactoryAddress'>) => {
+  return Object.values(obj).every((value) => !!value)
```

Comment:
Is this a replacement for `_.isEmpty()`?

The function name is `isContractNetworkConfig` but all this function does is check if all object's properties are truthy.

Reply:
true, the name doesn't make sense 👍 

The reason for the check is that if the contracts are not defined in the config service admin, the contractNetworks object comes back with all the properties defined but set to null: 
```
const contractAddresses: {
    safeSingletonAddress:  null;
    ...
}
```